core/src/models/Schedule.ts

```diff
@@ -224,11 +224,12 @@ export class Schedule extends LoggedModel<Schedule> {
     };
   }
 
-  async enqueueRun() {
+  async enqueueRun(triggeredBy?: string) {
     const run = await Run.create({
       creatorId: this.id,
       creatorType: "schedule",
       state: "running",
+      triggeredBy: triggeredBy,
     });
```

Comment:
I like this pattern a lot!
It might be interesting to to make `triggeredBy` be required.  Other options could be: 
* `manual` (user hit the button in the UI)
* `time` or `schcedule:check` (timer went off)

Reply:
Interesting!  Manual would also include users hitting the `/schedules/run` API too I would think.